diff --git a/node_modules/html2canvas/dist/html2canvas.js b/node_modules/html2canvas/dist/html2canvas.js
index 84cb30d..7ffdde1 100644
--- a/node_modules/html2canvas/dist/html2canvas.js
+++ b/node_modules/html2canvas/dist/html2canvas.js
@@ -1723,7 +1723,7 @@
             if (value.type === 18 /* FUNCTION */) {
                 var colorFunction = SUPPORTED_COLOR_FUNCTIONS[value.name];
                 if (typeof colorFunction === 'undefined') {
-                    throw new Error("Attempting to parse an unsupported color function \"" + value.name + "\"");
+                    return pack(0, 0, 0, 0);
                 }
                 return colorFunction(context, value.values);
             }
